poly-benches/benches/dense_multilinear.rs

```diff
@@ -0,0 +1,58 @@
+//! Benchmark for dense multilinear polynomial
```

Comment:
I think we should make the benchmark format style consistent between multilinear polynomials and `dense_polynomial` / `fft`. 

I prefer the approach of dense polynomial/fft since it reduces the boiler plate, and exposes flags for enabling particular benchmarks / using different fields.

Reply:
Make sense. I removed this inline doc